src/serde_arrow_primitive_array.hrl

```diff
@@ -0,0 +1,7 @@
+-record(primitive_array, {
+    type :: serde_arrow_type:arrow_type(),
+    len :: pos_integer(),
+    null_count :: pos_integer(),
+    validity_bitmap :: binary() | nil,
```

Comment:
```suggestion
    validity_bitmap :: binary() | undefined,
```

`undefined` is the convention in Erlang.

Reply:
Ok. Quite odd that Elixir does not follow that convention. In that case would all "nil" or "null" values in Erlang be `undefined`?

Reply:
How will we consume lists from Elixir? Should we accept both `undefined` and `nil` as null values?